### PR TITLE
Vmware zerto virtual replication vpn

### PR DESCRIPTION
**Pull request by RealAdaas**

### CODE REVIEW

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto-virtual-replication-customer-to-ovhcloud/guide.fr-fr.md [hunk 1]
```diff
@@ -19,9 +19,16 @@ Consultez notre guide « [Zerto entre deux datacenters OVHcloud](/pages/hosted_p
 - Les machines de réplication Zerto (VRA : Virtual Réplication Appliance) coté client et coté OVHCloud doivent pouvoir échanger entre elles sur les ports TCP 4007 et 4008
 - Les machines d’administration Zerto (ZVM : Zerto Virtual Manager) coté client et coté OVHCloud doivent pouvoir échanger entre elles sur les ports TCP 9081
 
+>[!primary]
+>
+> Pour une installation de Zerto opérationnelle
```
Comment (Kazarus4) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto-virtual-replication-customer-to-ovhcloud/guide.fr-fr.md]:
```suggestion
> Pour une installation de Zerto opérationnelle :
>
```

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto-virtual-replication-customer-to-ovhcloud/guide.fr-fr.md [hunk 2]
```diff
@@ -19,9 +19,16 @@ Consultez notre guide « [Zerto entre deux datacenters OVHcloud](/pages/hosted_p
...
+> [Binaire Zerto](https://www.zerto.com/myzerto/login/?redirect_to=%2Fmyzerto%2Fsupport%2Fdownloads%2F) (il est nécessaire d'avoir un compte)
```
Comment (Kazarus4) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto-virtual-replication-customer-to-ovhcloud/guide.fr-fr.md]:
```suggestion
> - [Téléchargez Zerto](https://www.zerto.com/myzerto/login/?redirect_to=%2Fmyzerto%2Fsupport%2Fdownloads%2F) (il est nécessaire d'avoir un compte).
```

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto-virtual-replication-customer-to-ovhcloud/guide.fr-fr.md [hunk 3]
```diff
@@ -19,9 +19,16 @@ Consultez notre guide « [Zerto entre deux datacenters OVHcloud](/pages/hosted_p
...
+> [Documentation d'installation Zerto](https://help.zerto.com/bundle/Install.VC.HTML/page/Installing_the_Zerto_Solution.htm)
```
Comment (Kazarus4) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto-virtual-replication-customer-to-ovhcloud/guide.fr-fr.md]:
```suggestion
> - [Documentation d'installation Zerto](https://help.zerto.com/bundle/Install.VC.HTML/page/Installing_the_Zerto_Solution.htm)
```

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto_virtual_replication_as_a_service/guide.fr-fr.md [hunk 4]
```diff
@@ -303,6 +303,8 @@ En cas d'incident majeur sur le site principal ou dans le cadre d'un exercice en
 >
 > Le failover en mode **LIVE** sur Zerto Replication se fait en considérant le site principal comme indisponible, il faut donc faire attention à la configuration réseau afin d'éviter notamment tout conflit d'adressage IP.
 >
+> De plus, dans le cas où les deux Private Cloud se situe dans 2 pays différents, il n'est pas possible de déplacer votre bloc IP d'un Private Cloud à un autre !
```
Comment (Kazarus4) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/zerto_virtual_replication_as_a_service/guide.fr-fr.md]:
```suggestion
> De plus, dans le cas où les deux Private Cloud se situent dans 2 pays différents, il est impossible de déplacer votre bloc IP d'un Private Cloud à un autre !
```

**Comment by tcpdumpfbacke** [comment]
Thank you @RealAdaas !

**Pull request merged**